x-pack/plugins/synthetics/server/synthetics_service/formatters/browser.ts

```diff
@@ -15,21 +15,16 @@ import { arrayFormatter, objectFormatter, stringToObjectFormatter } from './form
 export type BrowserFormatMap = Record<keyof BrowserFields, Formatter>;
 
 const throttlingFormatter: Formatter = (fields) => {
-  if (!fields[ConfigKey.IS_THROTTLING_ENABLED]) return false;
+  const value = fields[ConfigKey.THROTTLING_CONFIG];
+  const defaultThrottling = DEFAULT_BROWSER_ADVANCED_FIELDS[ConfigKey.THROTTLING_CONFIG];
+
+  if (value?.download === 0 && value?.upload === 0 && value?.latency === 0) return false;
+  if (value?.label === 'no-throttling') return false;
 
   return {
```

Comment:
This looks like the right formatter. Wondering if those formatters in common are even in use?